packages/core-pricing/plugins/product-price-coinbase-exchange.js

```diff
@@ -0,0 +1,71 @@
+import fetch from 'isomorphic-unfetch';
+import {
+  ProductPricingDirector,
+  ProductPricingAdapter,
+} from 'meteor/unchained:core-pricing';
+import { Countries } from 'meteor/unchained:core-countries';
+import Cache from '../utils/cache';
+
+const CACHE_PERIOD = 60 * 60 * 0.1; // 10 minutes
+const SUPPORTED_CURRENCIES = [
+  'BTC',
+  'ETH',
+  'XRP',
+  'USDT',
+  'BCH',
+  'BSV',
+  'LTC',
+  'EOS',
+  'BNB',
+  'XTZ',
+];
+const cache = new Cache(CACHE_PERIOD);
+
+const getFiatexchangeRateForCrypto = async (base, target) => {
+  const { data } = await cache.get(`${base}-${target}`, () =>
+    fetch(`https://api.coinbase.com/v2/exchange-rates?currency=${base}`, {
+      method: 'GET',
+    }).then((res) => res.json())
+  );
+  return data?.rates?.[target];
+};
+
+class ProductPriceCoinbaseExchange extends ProductPricingAdapter {
+  static key = 'shop.unchained.pricing.price-coinbase-exchange';
+
+  static version = '1.0';
+
+  static label = 'Convert EUR to crypto with curreny exchange rate';
```

Comment:
This plugin is not restricted to EUR isn't it? It's more like "Convert Default Currency to Crypto...."